CMakeLists.txt

```diff
@@ -38,6 +38,28 @@ IF ("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
   "begin a build in a different directory.")
 ENDIF()
 
+#
+# Setup CMAKE_BUILD_TYPE:
+#
+
+SET(CMAKE_BUILD_TYPE
+  "DebugRelease"
+  CACHE STRING
+  "Choose the type of build, options are: Debug, Release and DebugRelease."
+  )
```

Comment:
This is a variable in the cmake namespace. Isn't it defined by default?

Reply:
It is, but the default documentation says that only 'Debug' and 'Release' are allowed options. This way I overwrite the documentation with the additional option.  I copied this code from https://github.com/dealii/dealii/blob/master/cmake/setup_cached_variables.cmake do you want me to make the same change there too?

Reply:
I see. I trust @tamiko that it is ok to overwrite this variable.

And yes, it would of course be fantastic if you could make the same changes there as well!